src/test/java/com/o19s/es/ltr/logging/LoggingFetchSubPhaseTests.java

```diff
@@ -148,22 +146,19 @@ public void testLogging() throws IOException {
         }
     }
 
-    public void testBogusQuery() throws IOException {
```

Comment:
we should keep this test perhaps just expect that IllegalArgumentException is thrown?

Reply:
The issue is we cant even build a `BoostQuery` object anymore since that is the [one that throws](https://github.com/apache/lucene-solr/blob/master/lucene/core/src/java/org/apache/lucene/search/BoostQuery.java#L43) `IllegalArgumentException` so essentially we never even get to the code we want to test.